dive_core/command_hierarchy.cpp

```diff
@@ -798,6 +824,18 @@ bool CommandHierarchyCreator::OnIbEnd(uint32_t                  submit_index,
     m_state_tracker.PopEnableMask();
     DIVE_ASSERT(!m_ib_stack.empty());
 
+    // Setup root & range of shared children that this IB encompasses
```

Comment:
this is basically to mark the range of pm4s of this IB, right?

Reply:
Yup